app/store/middlewares/helpers.js

```diff
@@ -136,7 +136,7 @@ export function cleanUpState(payload, keepCurrent = false) {
 
     nextState.views.root = {
         // eslint-disable-next-line no-underscore-dangle
-        hydrationComplete: !nextState._persist,
+        hydrationComplete: true,
```

Comment:
Gave this a bit of thought.

```suggestion
        hydrationComplete: payload?.views?.root?.hydrationComplete || !nextState._persist,
```

By doing this on first run it will not mark it as true letting the rest of the process complete as it should, but when sent to the background it will keep the value previously set.